pubsub/cloud-client/subscriber_test.py

```diff
@@ -25,6 +25,8 @@
 PROJECT = os.environ['GCLOUD_PROJECT']
 TOPIC = 'subscription-test-topic'
 SUBSCRIPTION = 'subscription-test-subscription'
+SUBSCRIPTION_SYNC1 = 'subscription-test-subscription-sync1'
```

Comment:
Since these are synchronous and shouldn't step over each other, can we use the same subscription for both synchronous tests?

Reply:
They sometimes did step on each other because the tests are run in parallel.